src/shrines/SnapshotShrine.sol

```diff
@@ -106,6 +101,7 @@ contract SnapshotShrine is Stewarded, ReentrancyGuard {
     /// User actions
     /// -----------------------------------------------------------------------
 
+    // TODO: Update below notice to reflect structure
     /// @notice Offer ERC-20 tokens to the MerkleShrine and distribute them to Champions proportional
```

Comment:
IIUC we the structure is similar except we won't be calling it `MerkleShrine` - should we just change to `SnapshotShrine` or do we want a diff name

Reply:
@DaoDeCyrus

Reply:
Yup, Merkle => Snapshot @ParthSareen